src/tests/ftest/util/agent_utils.py

```diff
@@ -65,9 +152,21 @@ def run_agent(basepath, server_list, client_list=None):
     sessions = {}
     user = getpass.getuser()
 
-    # if empty client list, 'self' is effectively client
+    # if empty client list, 'test' is effectively client
     client_list = include_local_host(client_list)
 
+    # Create the DAOS Agent configuration yaml file to pass
+    # with daos_agent -o <FILE_NAME>
+    agent_yaml = os.path.join(test.tmp, "daos_agent.yml")
+    agent_config = DaosAgentConfig()
+    agent_config.get_params(test)
+    agent_config.hostlist.value = client_list
+
+    access_point = ":".join((server_list[0], str(agent_config.port)))
+    agent_config.access_points.value = access_point.split()
```

Comment:
Thanks. Looks good to me. I assume there will be a follow up PR making sure this will be the same value as the one used in daos_server.yml?

Reply:
Yes, that is DAOS-3730